.github/workflows/gh-workflow.yml

```diff
@@ -54,7 +54,7 @@ jobs:
 
   Push:
     name: Push container and chart to registry
-    if: github.ref == 'refs/heads/main'
+    if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')
```

Comment:
Any reason for adding this back? I removed it before since I removed the part where tags would trigger the workflow

Reply:
Nope, that was added by mistake. Good catch